lib/include/prjxray/xilinx/xc7series/utils.h

```diff
@@ -0,0 +1,34 @@
+
+#ifndef PRJXRAY_LIB_XILINX_XC7SERIES_UTILS_H_
+#define PRJXRAY_LIB_XILINX_XC7SERIES_UTILS_H_
+
+#include <prjxray/xilinx/xc7series/configuration.h>
+#include <prjxray/xilinx/xc7series/configuration_packet.h>
+#include <prjxray/xilinx/xc7series/frames.h>
+#include <prjxray/xilinx/xc7series/part.h>
+
+namespace prjxray {
+namespace xilinx {
+namespace xc7series {
+using PacketData = std::vector<uint32_t>;
+using BitstreamHeader = std::vector<uint8_t>;
+using ConfigurationPackage = std::vector<std::unique_ptr<ConfigurationPacket>>;
+
+PacketData createType2ConfigurationPacketData(const Frames::Frames2Data& frames,
```

Comment:
Comments?

Reply:
the same here, the comments are in the .cc file